gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
@notwaldorf By "this file" do you mean the service worker file, or the `dist/1.0/index.html`?

My point was that if we update `templates/head-meta.html`, that won't change the contents of `dist/1.0/index.html`